src/picongpu/include/plugins/hdf5/WriteSpecies.hpp

```diff
@@ -154,9 +154,9 @@ struct WriteSpecies
                  mapper
                  );
             counterBuffer.deviceToHost();
-            log<picLog::INPUT_OUTPUT > ("HDF5:  ( end ) copy particle to host: %1%") % Hdf5FrameType::getName();
```

Comment:
ah, now I get it. you aligned them according to the "depth" of their execution in the code :)